public/spec/ftp/spec.md

```diff
@@ -401,12 +401,12 @@ service ftp:Service on ftpListener {
 ```ballerina
 // Create a service object
 ftp:Service ftpListener = service object {
-    remote function onFileChange(ftp:WatchEvent & readonly event, ftp:Caller caller) {
+    remote method onFileChange(ftp:WatchEvent & readonly event, ftp:Caller caller) {
         // process event
     }
 };
 ```
-The remote function `onFileChange()` is invoked when the listener notices a file change in the FTP server. This function supports
+The remote method `onFileChange()` is invoked when the listener notices a file change in the FTP server. This function supports
```

Comment:
Only this change is relevant in this file.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/6013.